doc/sphinx/Pacemaker_Explained/reusing-configuration.rst

```diff
@@ -238,141 +228,133 @@ Reusing Parts of the Configuration
-   
-   === Using Tags in Constraints and Resource Sets ===
-   
-   Pacemaker itself only uses tags in constraints. If you supply a tag name
-   instead of a resource name in any constraint, the constraint will apply to
-   all resources listed in that tag.
-   
-   .Constraint using a tag
-   ====
-   [source,XML]
-   ----
-   <rsc_order id="order1" first="storage" then="all-vms" kind="Mandatory" />
-   ----
-   ====
-   
-   In the example above, assuming the +all-vms+ tag is defined as in the previous
-   example, the constraint will behave the same as:
-   
-   .Equivalent constraints without tags
-   ====
-   [source,XML]
-   ----
-   <rsc_order id="order1-1" first="storage" then="vm1" kind="Mandatory" />
-   <rsc_order id="order1-2" first="storage" then="vm2" kind="Mandatory" />
-   <rsc_order id="order1-3" first="storage" then="vm2" kind="Mandatory" />
-   ----
-   ====
-   
-   A tag may be used directly in the constraint, or indirectly by being
-   listed in a <<s-resource-sets,resource set>> used in the constraint.
-   When used in a resource set, an expanded tag will honor the set's
-   +sequential+ property.
+
+The same principle applies for ``meta_attributes`` and
+``instance_attributes`` as illustrated in the example below:
+
+.. topic:: Referencing attributes, options, and operations from other resources
+
+    .. code-block:: xml
+
+      <primitive id="mySpecialRsc" class="ocf" type="Special" provider="me">
+         <instance_attributes id="mySpecialRsc-attrs" score="1" >
+           <nvpair id="default-interface" name="interface" value="eth0"/>
+           <nvpair id="default-port" name="port" value="9999"/>
+         </instance_attributes>
+         <meta_attributes id="mySpecialRsc-options">
+           <nvpair id="failure-timeout" name="failure-timeout" value="5m"/>
+           <nvpair id="migration-threshold" name="migration-threshold" value="1"/>
+           <nvpair id="stickiness" name="resource-stickiness" value="0"/>
+         </meta_attributes>
+         <operations id="health-checks">
+           <op id="health-check" name="monitor" interval="60s"/>
+           <op id="health-check" name="monitor" interval="30min"/>
+         </operations>
+      </primitive>
+      <primitive id="myOtherlRsc" class="ocf" type="Other" provider="me">
+         <instance_attributes id-ref="mySpecialRsc-attrs"/>
+         <meta_attributes id-ref="mySpecialRsc-options"/>
+         <operations id-ref="health-checks"/>
+      </primitive>
+
+``id-ref`` can similarly be used with ``resource_set`` (in any constraint type),
+``nvpair``, and ``operations``.
+
+Tagging Configuration Elements
+##############################
+
+Pacemaker allows you to *tag* any configuration element that has an XML ID.
+
+The main purpose of tagging is to support higher-level user interface tools;
+Pacemaker itself only uses tags within constraints. Therefore, what you can
+do with tags mostly depends on the tools you use.
```

Comment:
Might as well update this section to reference crm_mon --resource/--node (probably in its own commit)